common-test.dotty/src/main/scala/org/scalatest/SharedHelpers.scala

```diff
@@ -1802,18 +1804,18 @@ object SharedHelpers extends Assertions with LineNumberHelper {
   def sortedSet[T](elements: T*): SortedSet[T] = {
     val orderMap = Map.empty[T, Int] ++ elements.zipWithIndex
     val comparator = orderMapComparator(orderMap)
-    implicit val ordering: Ordering[T] = new Ordering[T] {
-      def compare(x: T, y: T): Int = comparator.compare(x, y)
-    }
+    implicit val ordering: Ordering[T] = (new Ordering[T | Null] {
```

Comment:
**Java template class requires nullable type**
In order for operator overrides to be valid, java templates classes must use `T | Null` for the type. The nullness is then cast away using `asInstanceOf[ClassName[T]]`.